src/FieldType/Generator/EntityValidatorMetadataGenerator.php

```diff
@@ -65,7 +65,9 @@ public static function generate(FieldInterface $field, TemplateDir $templateDir,
             );
             $strings = [];
             foreach ($generatorConfig['entity']['validator'] as $assertion => $assertionOptions) {
+                $path = strpos("\"", $assertion)? 'new Assert\\': 'new \\';
```

Comment:
This has the potential to yield unexpected results because if the position of the \ can be on 0 and 0 will be evaluated as false. Explicitly check for false to make this work.

Also, it seems you are checking for " instead of \.

Reply:
indeed, so why was it working as expected??? 👽 👽

Reply:
[`ALIENS`](https://i.imgflip.com/l1yke.jpg)